### PR TITLE
UX/UI Improvements/Adjustments

### PR DESCRIPTION
**Pull request by philipbrito**
@RocketChat/android

This updates the attach icon, send icon and add a scroll indicator

videotogif_2017 07 26_16 55 37

### CODE REVIEW

**Comment by rafaelks** [comment]
Nice touch @filipedelimabrito! 👍 🚀

**Review by rafaelks — APPROVED** [review]

File: rocket-chat-android-widgets/src/main/res/layout/message_composer.xml [hunk 1]
```diff
@@ -1,62 +1,57 @@
...
-                    android:id="@+id/btn_submit"
-                    android:layout_width="48dp"
-                    android:layout_height="48dp"
-                    android:padding="8dp"
-                    app:srcCompat="@drawable/ic_send_black_24dp" />
-
-        </FrameLayout>
-    </LinearLayout>
-</LinearLayout>
+<android.support.constraint.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
```
Comment (luciofm) [inline rocket-chat-android-widgets/src/main/res/layout/message_composer.xml:2]: Can you reformat it following the new code style (https://github.com/RocketChat/Rocket.Chat.Android/blob/develop/CONDING_STYLE.md)?

File: rocket-chat-android-widgets/src/main/res/values/colors.xml [hunk 2]
```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="utf-8"?>
```
Comment (luciofm) [inline rocket-chat-android-widgets/src/main/res/values/colors.xml:1]: This one also

**Review by luciofm — CHANGES_REQUESTED** [review]
Just some styling issues

File: rocket-chat-android-widgets/src/main/res/layout/message_composer.xml (hunk 1 above)
Comment (philipbrito) [reply rocket-chat-android-widgets/src/main/res/layout/message_composer.xml:2]: Of course! Already done! :)

File: rocket-chat-android-widgets/src/main/res/values/colors.xml (hunk 2 above)
Comment (philipbrito) [reply rocket-chat-android-widgets/src/main/res/values/colors.xml:1]: Done @luciofm !

**Review by luciofm — APPROVED** [review]
Thanks :)

**Review by luciofm — CHANGES_REQUESTED** [review]
The build is broken...

```
:rocket-chat-android-widgets:compileReleaseJavaWithJavac
:rocket-chat-android-widgets:compileReleaseJavaWithJavac - is not incremental (e.g. outputs have changed, no previous execution, etc.).
/home/travis/build/RocketChat/Rocket.Chat.Android/rocket-chat-android-widgets/src/main/java/chat/rocket/android/widget/message/MessageFormLayout.java:167: error: cannot find symbol
    composer.findViewById(R.id.sendImageButton).setEnabled(enabled);
                              ^
  symbol:   variable sendImageButton
  location: class id
```

**Review by luciofm — APPROVED** [review]

**Pull request merged**